Source/DataSources/ModelGraphics.js

```diff
@@ -104,6 +107,15 @@ define([
         maximumScale : createPropertyDescriptor('maximumScale'),
 
         /**
+         * Get or sets the boolean Property specifying whether textures
+         * may continue to stream in after the model is loaded.
+         * {@link ModelGraphics#incrementallyLoadTextures}.
```

Comment:
This link line is unnecessary.  I assume it's copied from `maximumScale` above, but that's linking to `minimumPixelSize`, this is linking to itself.